bigframes/operations/semantics.py

```diff
@@ -527,6 +527,109 @@ def search(
 
         return typing.cast(bigframes.dataframe.DataFrame, search_result)
 
+    def sim_join(
+        self,
+        other,
+        left_on: str,
+        right_on: str,
+        model,
+        top_k: int = 3,
+        score_column: Optional[str] = None,
+        max_rows: int = 1000,
+    ):
+        """
+        Joins two dataframes based on the similarity of the specified columns.
```

Comment:
To provide more context and clarity for users, let's enhance the description of the`sim_join` method with details about its implementation and scalability. Please refer to `agg` method. Thanks

Reply:
SG. Done!